cmd/util.go

```diff
@@ -20,11 +21,35 @@ var (
 	}
 )
 
-func handleError(err error, context string) (errmsg string) {
+func printWarn(msg string) {
+	fmt.Fprintf(os.Stderr, "[%s] %s\n",
+		colors[check.WARN].Sprintf("%s", check.WARN),
+		msg,
+	)
+}
+
+func printfWarn(msg string) string {
```

Comment:
This is a small thing but I think for readability it would be better to call this function `sprintWarn`, to be clearer about the fact this is returning a string (that would be more consistent with the difference between printf and sprintf)